Report.java

```diff
@@ -202,4 +205,16 @@ private void printSpriteCounts(PrintWriter printW, int i)
             printW.println();
         }
     }   
+
+    /**
+     * Method to get the current datetime for report.
+     *
+     * @return formatted datetime
+     */
+    public String getReportDateTime()
+    {
+        DateFormat dateFormat = new SimpleDateFormat("yyyyMMddHHmmss");
```

Comment:
I tried it with dashes, but it causes the filename to get pretty long. Symbols can also be tricky in filenames depending on what OS you're on. @KaraBeason @caldwellc1 what do you guys think about leaving it together like this, just for a unique ID for the report file, and making it readable on the report output?